File: easy/merge_sorted_arr#88.c
void merge(int* nums1, int nums1Size, int m, int* nums2, int nums2Size, int n){
int i2=0;
    //printf("%d %d\n",nums1Size,nums2Size);
for(int i=0;i<nums1Size;i++)
{
    if(i2<nums2Size && nums1[i]>nums2[i2])
    {
        for(int j=nums1Size-1;j>i;j--)
        {
            nums1[j]=nums1[j-1];
            //printf("%d\n",nums1[j]);
        }
        //printf("a");
        nums1[i]=nums2[i2];
        //printf("%d\n",nums1[i]);
        i2++;
    }
    else if(nums1[i]==0 && i>=nums1Size-nums2Size+i2)
    {
        //printf("b");
        nums1[i]=nums2[i2];
        i2++;
    }
    
}
}

File: easy/merge_sorted_arr#88.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        y=0
        for x in range(len(nums1)):
            if(y<len(nums2) and nums1[x]>nums2[y]):
                for z in range(len(nums1)-2,x-1,-1):
                    nums1[z+1]=nums1[z]
                nums1[x]=nums2[y]
                y+=1
            if(x-y==m and nums1[x]==0):
                for z in range(x,len(nums1)):
                    nums1[z]=nums2[y]
                    y+=1
                break
